BREAKING.md

```diff
@@ -18,10 +18,14 @@ There are a few steps you can take to write a good change note and avoid needing
 
 ## 0.58 Upcoming changes
```

Comment:
@heliocliu I would recommend changing this to not say 0.58, maybe "Upcoming changes (ETA 0.59)" or something.  Or just "Upcoming changes" and as part of the release clear the section out every time...?

Reply:
I definitely wanted to avoid the step of needing to clear them out, but it's true that having a version associated with the upcoming change has been confusing for people :/ But yeah the ease of understanding probably outweighs needing to delete stuff occasionally there.

Reply:
Why would you need to clear them out?